brainstormforce/astra
# AST-934 - Setting up Laravel-Mix in Astra

### Description
- https://github.com/laravel-mix/laravel-mix
- Setting up Laravel-Mix in Astra

### To-Do Setup
- yarn install
- yarn run build

### Advantages
- Internal directory package.json & node setup can be removed
- Laravel-Mix's JS, SCSS, CSS, BrowserSync helpers can be used

### Things to do before that
- Faced problem with npm but it worked with yarn
- Need to check it first, then if not sync with npm then code migration needs to be done
- Facing errors in console regarding JSX - https://share.bsf.io/Qwu5Jqbx (There can be many)

### Types of changes
- Improvement

### How has this been tested?
- Check customizer build
- Check metabox build JS

### Checklist:
